string mqtt_topic
string ros_topic
string msg_type
---
bool success